app/src/main/res/values/strings.xml

```diff
@@ -231,6 +231,7 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_maxspeed_answer_sign">"A sign"</string>
     <string name="quest_maxspeed_answer_zone2">"It's within a (slow) zone"</string>
     <string name="quest_maxspeed_answer_noSign2">"No sign, defaults apply"</string>
+    <string name="quest_maxspeed_answer_nsl">"National Speed Limit"</string>
```

Comment:
Should this be translated at all? F.e. if I am on a vacation in England, should it say "Nationale Höchstgeschwindigkeit" for me or should it stay untranslated? I think maybe it should stay untranslated, in this case it should go into `untranslatableStrings.xml`

Reply:
Hmm, possibly untranslated. You'll never see "National Speed Limit" on a sign anywhere, you'll just see the sign. But it is also a technical, official term.